draft-ietf-core-new-block.xml

```diff
@@ -859,9 +837,14 @@ missing-block-number = uint
     </section>
 
     <section anchor="cc" title="Congestion Control">
-      <t>The transmission of the payloads of a body either SHOULD all be
-      Confirmable or all be Non-confirmable. This is meant to simplify the
-      congestion control procedure.</t>
+      <t>The transmission of the payloads of a body over UDP or DTLS connection
```

Comment:
```suggestion
      <t>The transmission of the payloads of a body over an unreliable transport      
```

Reply:
I thought about using unreliable, but then CoAP supports reliable transmission when using CON.  Hence my text, as you could argue CoAP is a reliable transport when using UDP.

Reply:
CoAP is not a transport.

Reply:
OK - Change included.